### ISSUE TITLE
labeled icon <Button /> not rendered as <button>

### ISSUE THREAD

**Issue by Chris-R3** [Semantic-Org/Semantic-UI-React#596]
### Steps to Reproduce

I'd like to render a `labeled  icon button` with `type='submit'` with following jsx code:

``` jsx
<Button type='submit' icon='save' labeled content='save' />
```
### Expected Behavior

``` html
<button class="ui labeled icon button" type="submit">
  <i class="save icon"></i>
  save
</button>
```
### Actual Behavior

``` html
<div type="submit" class="ui labeled icon button" tabindex="0">
  <i class="save icon"></i>
  save
</div>
```
### Test Case

http://codepen.io/Chlris-R3/pen/xEXBzK?editors=0010

Is this a bug or do I need to change my code somehow? I also tried adding `as='button'` with the same result.

**Comment by levithomason** [comment]
Confirmed bug, thanks much.  @yesmin is preparing the fix in #597.

**Comment by Chris-R3** [comment]
much appreciated
keep up the good work :heart:

**Comment by levithomason** [comment]
Fixed in #597 and released in `stardust@0.52.5`
